src/acl_types.rs

```diff
@@ -0,0 +1,115 @@
+use std::time::Duration;
+
+use serde::{self, Deserialize, Serialize};
+
+/// Information related ACL token.
+/// See https://developer.hashicorp.com/consul/docs/security/acl/tokens for more information.
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "PascalCase")]
+pub struct ACLToken {
+    /// Unique ID
+    #[serde(rename = "AccessorID")]
+    pub accessor_id: String,
+    /// Secret for authenticatioIDn
```

Comment:
Please fix all occurrences of this typo
```suggestion
    /// Secret for authentication
```